crates/warp-contracts/warp-contracts-core/src/evolve_state.rs

```diff
@@ -0,0 +1,17 @@
+use serde::Serialize;
+
+#[derive(Serialize, Debug)]
+pub struct EvolveState {
+    pub can_evolve: bool,
```

Comment:
will this map properly to fields in json state, i.e. 'canEvolve' and 'evolve'?

Reply:
No, it won't. This is supposed to be fetched by SDK WasmHandlerApi and process there. I didn't see reasons to make these field names compatible with our usual State fields. But of course it can be done.